src/styles/styles.js

```diff
@@ -716,13 +716,11 @@ const styles = {
     textInputDesktop: addOutlineWidth({}, 0),
 
     secureInputEyeButton: {
```

Comment:
NAB I know this is not your code, but i feel like we should rename this to something more indicative of what it does, e.g. `secureInputShowPasswordButton`

Reply:
Done